armi/reactor/reactors.py

```diff
@@ -291,22 +292,21 @@ def powerMultiplier(self):
         return geometry.SYMMETRY_FACTORS[self.symmetry]
 
     @property
-    def lib(self):
+    def lib(self) -> Optional[xsLibraries.IsotxsLibrary]:
```

Comment:
Does this have to be IsotxsLibrary specifically? Could it be the base... oh, nvm `_XSLibrary` is priavate?

Reply:
Yes it's private.